RevenueCatUI/CustomerCenter/Data/CustomerCenterPurchases.swift

```diff
@@ -82,6 +82,10 @@ final class CustomerCenterPurchases: CustomerCenterPurchasesType {
         try await Purchases.shared.syncPurchases()
     }
 
+    func virtualCurrencies() async throws -> VirtualCurrencies {
+        return try await Purchases.shared.virtualCurrencies(forceRefresh: true)
```

Comment:
We always want to hard refresh VCs in the Customer Center so that we don't accidentally show stale balances